admin/experiment_edit.php

```diff
@@ -120,6 +120,7 @@
         foreach ($formvarnames as $fvn) {
             if (!isset($edit[$fvn])) $edit[$fvn]="";
         }
+	$edit['access_restricted']=$settings['default_experiment_restriction'];
```

Comment:
Sorry, I messed this up. I applied the change in my local branch, but then committed with --amend, which then did not let me push the update branch to the remote repository. So I deleted the current branch on the remote repository and uploaded it as a new one. I guess I now have to open a new pull request for that new branch?

Reply:
Ben, after you committed with --amend param, you can force-push (push with -f param) to the same branch and this will update this pull request accordingly.